README.md

```diff
@@ -93,6 +93,9 @@ Key | Description | Default value
 | run-before | Script that is executed before the actual command | *no default*
 | run-after | Script that is executed after the actual command | *no default*
 | alias | Allows to set short aliases for long commands<br>`my_command: "--ri --with-docker-mount --image=my-image --image-version=my-tag -E my-script.py"` | *no default*
+| auto-update | Toggles the auto update check. Will only perform the update after the delay | true
```

Comment:
I would say `Will only check for update` instead of `Will only perform the update`

Reply:
The reason for the nuance is that there are 2 things happening when auto update is enabled: 
1- check if we need to make an update (depending on de delay)
2- actually perform the update
We can enable auto-update form the config and it will run. But it wont perform the update unless the delay is passed. (Makes sense ?)